ImageItem/ImageItem.module.less

```diff
@@ -0,0 +1,30 @@
+// ImageItem.module.less
+//
+
+@import '../styles/skin.less';
+@import '../styles/mixins.less';
+
+.imageItem {
+  	.applySkins({
+		border: @agate-imageitem-border-width solid @agate-imageitem-border-color;
+
+		.focus({
+			&::before {
```

Comment:
Don't we need a `content: ""`?

Reply:
It seems that the solution works even without this property and that's why I removed it. I will add it back.